### ISSUE TITLE
Outdated email address in output

### ISSUE THREAD

**Issue by tom-englert** [Microsoft/CodeContracts#232]
Today I got this error message: 
```
CodeContracts: TomsToolbox.Wpf: Internal error: Method TomsToolbox.Wpf.Converters.ConfirmedCommandConverter+CommandProxy.#ctor(TomsToolbox.Wpf.Converters.ConfirmedCommandConverter,System.Windows.Input.ICommand) was found in the cache but deserialization failed.
CodeContracts: TomsToolbox.Wpf: The reason why the deserialization failed is Unable to find type ICommand
CodeContracts: TomsToolbox.Wpf: We remove the incorrect entry from the cache and we replace it with a new one. 
If the error persists, please report the tool output to logozzo@microsoft.com
```
**logozzo@microsoft.com** is probably no longer valid.
We should replace all email addresses with a link to the github issues page.

**Comment by tom-englert** [comment]
@mike-barnett there is some error reporting workflow in the ContractAdornments service provider that sends emails to 
```const string CrashMailRecepients = "ccixfb@microsoft.com";```
Is this still a valid address? Will those mails be processed?

**Comment by mike-barnett** [comment]
Dang, I forgot about that. No, we've deleted that alias. What do you think we should replace that with? Is there a way to have it automatically get added to an issue in GitHub?

**Comment by sharwell** [comment]
@mike-barnett We could always [generate a link like this](https://github.com/Microsoft/CodeContracts/issues/new?title=ContractAdornments+Crash+Report&body=Place+additional+details+here...%0A%0A--%0AException+details:%0A```%0ATODO%0A```).

**Comment by tom-englert** [comment]
@mike-barnett I've already sent a couple of crash reports :grin:

**Comment by mike-barnett** [comment]
@sharwell: can we make it so that it just adds to an existing issue? (Just because there probably will be a lot of them and hopefully they share a common root cause.) @tom-Englert: Unlikely that anyone has seen them...

**Comment by sharwell** [comment]
@mike-barnett At least initially I think I would prefer closing duplicates to reopening the same issue over and over.

**Comment by sharwell** [comment]
@mike-barnett I'm not sure either. Automated issue reports will be a new one for a project I'm on.

**Comment by tom-englert** [comment]
@mike-barnett would it be possible to reactivate ccixfb@microsoft.com as forwarder to us? I get this message at least once a day, but all the information will be lost if the mail gets trashed.

**Comment by mike-barnett** [comment]
I don't think that an internal alias can forward things to an outside address. @SergeyTeplyakov, do you know?

**Comment by SergeyTeplyakov** [comment]
@mike-barnett I never heard about it. I'll ask someone about this...

**Comment by sharwell** [comment]
I don't think users will expect crash dumps sent to `...@microsoft.com` to ever be sent to people not employed by Microsoft, or at least covered by fairly strict confidentiality agreements.

**Comment by mike-barnett** [comment]
Agreed. I would really prefer for it to be *not* a Microsoft.com email address.

**Comment by tom-englert** [comment]
@mike-barnett I thought this just to be an intermediate solution, to catch the error information of the existing versions. I get this "Do you want to send a crash report" at least once a day, and there is no feedback to the user to which email this information will be sent. It would be sufficient to forward the mails to you.
Of course we should replace this with a proper solution for the next version.
